jbpm-workitems/jbpm-workitems-rest/src/test/java/org/jbpm/process/workitem/rest/BasicAuthRestWorkItemHandlerTest.java

```diff
@@ -104,6 +107,7 @@ public void setClientApiVersion() {
 
     @Test
     public void testGETOperation() {
+        System.setProperty(RESTWorkItemHandler.USE_SYSTEM_PROPERTIES, "true");
```

Comment:
This is not testing anything. Just covering the line you added. remove this.

Reply:
This is added for coverage and for testing that, if property is enable and there is not any system property added, this method behaves as beforee. 
So, I think is a very productive addition. I vote for keeping it

Reply:
No it is not. 
1) You are modifying the behaviour of the test and make it dependent of the environment (so it is out of control)
2) You are not testing properly if the new property is using the system properties or not. e.g. one example would be to set a wrong address default adress (unreachable) and a proxy address setting in the system properties. then see if the system is taking or not what you are doing running a get operation against the proxied server.
This is not a proper test. Please remove and add a proper test.

Reply:
Ok, I will remove and define a specific case.

Reply:
Hi, you need to try to test this as well. otherwise looks good.

Reply:
Additional test case using http.proxy system property has been added.